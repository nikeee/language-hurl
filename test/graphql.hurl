POST https://example.org/graphql
```graphql
{
  human(id: "1000") {
    name
    height(unit: FOOT)
  }
}
```
HTTP 200
