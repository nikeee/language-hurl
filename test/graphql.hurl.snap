>POST https://example.org/graphql
#^^^^ source.hurl keyword.control.method.hurl
#    ^ source.hurl
#     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl string.quoted.url.hurl string.quoted.url.hurl
>```graphql
#^^^^^^^^^^ source.hurl meta.embedded.block.graphql.hurl
>{
#^^ source.hurl meta.embedded.block.graphql.hurl source.graphql
>  human(id: "1000") {
#^^^^^^^^^^^^^^^^^^^^^^ source.hurl meta.embedded.block.graphql.hurl source.graphql
>    name
#^^^^^^^^^ source.hurl meta.embedded.block.graphql.hurl source.graphql
>    height(unit: FOOT)
#^^^^^^^^^^^^^^^^^^^^^^^ source.hurl meta.embedded.block.graphql.hurl source.graphql
>  }
#^^^^ source.hurl meta.embedded.block.graphql.hurl source.graphql
>}
#^^ source.hurl meta.embedded.block.graphql.hurl source.graphql
>```
#^^^ source.hurl meta.embedded.block.graphql.hurl
>HTTP 200
#^^^^ source.hurl constant.language.version.hurl
#    ^^^^ source.hurl constant.numeric.status.hurl
>